app/views/posts/show.html.erb

```diff
@@ -1,16 +1,25 @@
 <a href="<%= user_posts_path() %>">
-    <button id="back-btn">⬅</button>
+  <button id="back-btn">⬅</button>
 </a>
 <h2>Post <%= "#" + @post.id.to_s %> by <%= @user.name %></h2>
 <div class='post-interactions'>
-    <span>Comments: <%= @post.comments_counter %>,</span>
-    <span>Likes: <%= @post.likes_counter %></span>
+  <span>Comments: <%= @post.comments_counter %>,</span>
+  <span>Likes: <%= @post.likes_counter %></span>
 </div>
+<% if current_user.present? %>
+  <div class="post-actions">
+    <%= button_to "Like", like_user_post_path(@user, @post), method: :post, remote: true %>
```

Comment:
- [ ] I get an error when I try to create a like with your form. Please make sure when a user clicks a like, it is incremented on the post.
![image](https://github.com/AbdelazizSoliman/Blog-App/assets/45460192/942a2389-c3ce-4db8-89c8-49c007aa2410)